B2b phase 1

tamigoodall opened the pull request:
<!---
Notes:
- Your PR should conform to our [Contributing Guidelines](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md)
- If applicable, add details to the [update feed](https://github.com/auth0/docs/tree/master/updates)
- Make sure your PR gets deployed to a Heroku [Review App](https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#review-apps) without errors
- Please include a short description
- If your PR is a work in progress, title it [DO NOT MERGE]
--->

mostekcm committed:
Add first version of B2B Authentication:

mostekcm committed:
removed duplicat paragraph

mostekcm committed:
fix missing trailing line

mostekcm committed:
clean commit of sidebar changes

mostekcm committed:
clean commit of sidebar changes

rkhoriander committed:
Merge pull request #7772 from mostekcm/b2b-phase-1-authentication

B2b phase 1 authentication

rkhoriander committed:
Fix title in metadata

tamigoodall committed:
Merge pull request #7775 from mostekcm/b2b-phase-1-sidebar-and-intro

clean commit of sidebar changes

mostekcm committed:
B2B Phase 2 and add m2m to both B2C and B2B

mostekcm committed:
B2B Phase 2 and add m2m to both B2C and B2B

mostekcm committed:
Update articles/architecture-scenarios/implementation/b2c/b2c-authentication.md

Co-Authored-By: Peter Fernandez <34482303+PeterGFernandez@users.noreply.github.com>

mostekcm committed:
Apply suggestions from code review

Peter's changes

Co-Authored-By: Peter Fernandez <34482303+PeterGFernandez@users.noreply.github.com>

tamigoodall committed:
Added b2b-architecture

Added b2b-architecture.md
Updated _includes/_architecture/_introduction.md to include b2b platform info

tamigoodall committed:
Create b2b-authorization.md

Created b2b-authorization.md

tamigoodall committed:
Merge pull request #7798 from auth0/b2b-architecture-phase-1

B2b architecture phase 1

tamigoodall committed:
Update _application-integration.md

Added missing paragraph

tamigoodall committed:
Update _application-integration.md

Fixed code in paragraph

tamigoodall committed:
Merge pull request #7799 from auth0/b2b-authorization-phase-1

Create b2b-authorization.md

tamigoodall committed:
Created b2b-profile-mgmt

Created b2b-profile-mgmt.md file
Resolved missed review comments from PR-7798 in architecture files
Changed "should" to "can"

tamigoodall committed:
Update _introduction.md

Changed N.B. to "All"

tamigoodall committed:
Create b2b-branding.md

Created b2b-branding.md

mostekcm committed:
B2B phase 1 & 2 provisioning

tamigoodall committed:
Create b2b-branding.md

Created b2b-branding.md file
Added b2b info to the _introduction.md file

tamigoodall committed:
Create b2b-deployment.md

Created b2b-deployment.md

tamigoodall committed:
Create b2b-logout.md

Created b2b-logout.md

tamigoodall committed:
Create b2b-qa.md

Created b2b-qa.md

tamigoodall committed:
Create b2b-operations.md

Created b2b-operations.md

tamigoodall committed:
Fixed includes

Fixed includes

tamigoodall committed:
Fixed include

Fixed include

tamigoodall committed:
Update b2b-provisioning.md

Minor changes

tamigoodall committed:
Merge branch 'master' into b2b-phase-1

tamigoodall committed:
Test checklist links

Test Analyze checklist link

tamigoodall committed:
Test Add planning pdf

Test Added planning pdf

rkhoriander committed:
Update checklists.md

tamigoodall committed:
Testing links to PDFs

Testing links to PDFs and xlsw

tamigoodall committed:
Merge branch 'test-checklists' of github.com:auth0/docs into test-checklists

tamigoodall committed:
Update b2c-architecture.md

Changed back to old link to find failure

tamigoodall committed:
Delete B2C-Project-Planning.pdf

PDF file too big - deleting it

tamigoodall committed:
Added icon

Added icon

tamigoodall committed:
Added all other spreadsheet links

Added icons
Made link H2s
Added other spreadsheets

tamigoodall committed:
Update checklists.md

Changed instructions

tamigoodall committed:
Merge pull request #7816 from auth0/test-checklists

Add checklist files and new links

tamigoodall committed:
Apply suggestions from code review

Co-Authored-By: Carlos Mostek <carlos@monthlymealplanner.com>

tamigoodall committed:
Merge pull request #7806 from auth0/b2b-qa-phase-1

Create b2b-qa.md

tamigoodall committed:
Merge pull request #7805 from auth0/b2b-logout-phase-1

Create b2b-logout.md

tamigoodall committed:
Merge pull request #7804 from auth0/b2b-deployment-phase-1

Create b2b-deployment.md

tamigoodall committed:
Merge pull request #7808 from auth0/b2b-operations-phase-1

Create b2b-operations.md

tamigoodall committed:
Apply suggestions from code review

Co-Authored-By: Carlos Mostek <carlos@monthlymealplanner.com>

tamigoodall committed:
Merge pull request #7802 from auth0/b2b-branding-phase-1

B2b branding phase 1

tamigoodall committed:
Apply suggestions from code review

Co-Authored-By: Carlos Mostek <carlos@monthlymealplanner.com>

tamigoodall committed:
Merge pull request #7801 from auth0/b2b-profile-mgmt-phase-1

B2b profile mgmt phase 1

tamigoodall committed:
Unresolved review comments

Added Peter's review comments from 2 other merged PRs
7808
7805

tamigoodall committed:
Clarifying tweaks

Clarifying tweaks

tamigoodall committed:
Update _introduction.md

Minor edit

tamigoodall committed:
More PeterF changes

More PeterF changes to Branding content

tamigoodall committed:
Update _introduction.md

Added change from PeterF to the Architecture introduction

tamigoodall committed:
Update b2b-architecture.md

Unresolved comments from PeterF to PR 7801

tamigoodall committed:
Merge pull request #7807 from mostekcm/b2b-phase-1-and-2-provisioning

B2B phase 1 & 2 provisioning

mostekcm committed:
Move M2M to authorization

tamigoodall committed:
Merge pull request #7819 from auth0/b2b-phase-1-review-comments

Unresolved review comments

tamigoodall committed:
Added new planning guide links

Added new smaller PDFs to /media
Changed _b2b-planning to _planning and added platform conditions
Changed links to all implementation section docs to include new _planning.md link

tamigoodall committed:
Smaller PDFs

Smaller PDFs

tamigoodall committed:
Merge branch 'b2b-phase-1' into b2b-phase-2-authentication

tamigoodall committed:
Merge pull request #7786 from mostekcm/b2b-phase-2-authentication

B2B Phase 2 and add m2m to both B2C and B2B

tamigoodall committed:
Update _introduction.md

tamigoodall committed:
Update b2b-authentication.md

tamigoodall committed:
Merge branch 'b2b-phase-1' into b2b-phase-2-authentication-take-2

tamigoodall committed:
Merge pull request #7821 from mostekcm/b2b-phase-2-authentication-take-2

B2b phase 2 authentication take 2

tamigoodall committed:
Update B2B-Project-Planning.pdf

New updated smaller file

tamigoodall committed:
Merge branch 'b2b-phase-1' into planning-guides-v2

tamigoodall committed:
Merge pull request #7822 from auth0/planning-guides-v2

Added new planning guide links

mostekcm commented on articles/architecture-scenarios/_includes/_authentication/_introduction.md:
```diff
@@ -1,28 +1,38 @@
-In order to provide services to your users, you must be able to identify who those users are. This process is called User  Authentication. There are a number of ways to perform authentication of a user - via social media accounts, username & password, <dfn data-key="passwordless">passwordless</dfn> - and it's often recommended that you go beyond a first factor for authenticating the user by addin ...
+In order to provide services to your users, you must be able to identify who those users are. This process is called User  Authentication. There are a number of ways to perform authentication of a user - via social media accounts, username & password, passwordless - and it's often recommended that you go beyond a first factor for authenticating the user by adding a second factor as well (a.k.a. M ...
```
Do we want to have that data-key included?

tamigoodall replied:
I'll add the glossary popups later

mostekcm commented on articles/architecture-scenarios/_includes/_authentication/_introduction.md:
```diff
@@ -1,28 +1,38 @@
 If you have an existing legacy identity store, you’ll also want to see [User Migration](/architecture-scenarios/implementation/${platform}/${platform}-provisioning#user-migration). This section discusses the advantages of migrating to Auth0’s managed identity storage in terms of safety and security.
 
-For customer-facing applications, <dfn data-key="openid">[OpenID Connect (OIDC)](/protocols/oidc)</dfn> is the most frequently used industry standard protocol, and OIDC has first-class citizen support in Auth0. Auth0 provides support for various different approaches for integrating various different applications, so you'll want to see the section on [application integration](#application-integrat ...
+For customer facing applications, OpenID Connect ([OIDC](/protocols/oidc)) is the most frequently used industry standard protocol, and OIDC has first-class citizen support in Auth0. Auth0 provides support for various different approaches for integrating various different applications, so you'll want to see the section on [application integration](#application-integration) for the information you' ...
```
do we need the data-key included?

tamigoodall replied:
No - I'll add glossary stuff later

mostekcm commented on articles/architecture-scenarios/implementation/b2b/b2b-authentication.md:
```diff
@@ -0,0 +1,133 @@
+"Bring your own identity" is a great feature to provide, but if you don't support this from day one, and sometimes even if you do, you may have an organization that wants to switch to their own IDP after already having used the application for a while.  You will need a way to [Link User Accounts](https://auth0.com/docs/link-accounts) (a.k.a. Account Linking) to provide an effective way of associa ...
+:::
+
+## Machine-to-Machine (M2M) authentication
```
```suggestion
```

tamigoodall replied:
Done

mostekcm commented on articles/architecture-scenarios/implementation/b2b/b2b-authentication.md:
```diff
@@ -0,0 +1,133 @@
+:::
+
+## Machine-to-Machine (M2M) authentication
+
```
```suggestion
```

mostekcm commented on articles/architecture-scenarios/implementation/b2b/b2b-authentication.md:
```diff
@@ -0,0 +1,133 @@
+
+## Machine-to-Machine (M2M) authentication
+
+<%= include('../../_includes/_authentication/_m2m.md', { platform: 'b2b' }) %>
```
```suggestion
```

tamigoodall replied:
Done

mostekcm commented on articles/architecture-scenarios/implementation/b2b/b2b-authentication.md:
```diff
@@ -0,0 +1,133 @@
+## Machine-to-Machine (M2M) authentication
+
+<%= include('../../_includes/_authentication/_m2m.md', { platform: 'b2b' }) %>
+
```
```suggestion
```

mostekcm commented on articles/architecture-scenarios/implementation/b2c/b2c-authentication.md:
```diff
@@ -39,13 +39,17 @@ useCase:
 
 <%= include('../../_includes/_authentication/_social-authentication.md', { platform: 'b2c' }) %>
 
+## Machine-to-Machine (M2M) authentication
```
```suggestion
```

tamigoodall replied:
Done

mostekcm commented on articles/architecture-scenarios/implementation/b2c/b2c-authentication.md:
```diff
@@ -39,13 +39,17 @@ useCase:
 <%= include('../../_includes/_authentication/_social-authentication.md', { platform: 'b2c' }) %>
 
+## Machine-to-Machine (M2M) authentication
+
```
```suggestion
```

tamigoodall replied:
Done

mostekcm commented on articles/architecture-scenarios/implementation/b2c/b2c-authentication.md:
```diff
@@ -39,13 +39,17 @@ useCase:
 
+## Machine-to-Machine (M2M) authentication
+
+<%= include('../../_includes/_authentication/_m2m.md', { platform: 'b2c' }) %>
```
```suggestion
```

tamigoodall replied:
Done

mostekcm commented on articles/architecture-scenarios/implementation/b2c/b2c-authentication.md:
```diff
@@ -39,13 +39,17 @@ useCase:
+## Machine-to-Machine (M2M) authentication
+
+<%= include('../../_includes/_authentication/_m2m.md', { platform: 'b2c' }) %>
+
```
```suggestion
```

tamigoodall replied:
Done

mostekcm commented on articles/architecture-scenarios/_includes/_authentication/_m2m.md:
```diff
@@ -0,0 +1,14 @@
+* A separate API the supports privileged access (e.g. the API is not exposed to users directly, but instead to a backend only).
+* In certain microservice architectures, where some API layers need to communicate to other API layers without a user involvement, or after a user token has expired.
+* A privileged API that may need to be called before a user has authenticated (i.e. from a rule or custom DB script in your Auth0 tenant)
+
```
This whole file needs to be removed...

mostekcm replied:
it was replaced with _authorization/_m2m.md

tamigoodall replied:
Done

mostekcm commented on articles/architecture-scenarios/_includes/_profile-mgmt/_introduction.md:
```diff
@@ -16,17 +16,24 @@ The Normalized User Profile is updated from the identity provider during login,
 Alternatively, the Auth0 Dashboard can be used to [manage aspects of a user’s profile](users/guides/manage-users-using-the-dashboard). Managing a user’s profile via the Auth0 Dashboard is more of an administrative provision and **should not** be used for self-serviced profile management in a production environment. However, the interface provided by the Dashboard can be extremely useful during de ...
+
+<% if (platform === "b2b") { %>
+* If you need to provide a way for your customers to have an administrator that can manage their own users when they are storing those credentials in your system, you can either build something yourself or use an Auth0 Extension. See [Admin Portal](#admin-portal) for more details. 
```
```suggestion
If you need to provide a way for your customers to have an administrator that can manage their own users when they are storing those credentials in your system, you can either build something yourself or use an Auth0 Extension. See [Admin Portal](#admin-portal) for more details. 
```

mostekcm commented on articles/architecture-scenarios/_includes/_provisioning/_introduction.md:
```diff
@@ -6,10 +6,23 @@ Whilst Auth0 supports numerous workflows, web based workflows using Auth0 [Unive
 * Can my users sign up using their existing social - e.g. Facebook or Google - accounts?
+<% } %>
+<% if (platform==="b2b") { %>
+* Can your users sign up using their organization's identity provider?
```
```suggestion
* Can my users sign up using their organization's identity provider?
```

tamigoodall replied:
Nope

mostekcm commented on articles/architecture-scenarios/_includes/_provisioning/_introduction.md:
```diff
@@ -6,10 +6,23 @@ Whilst Auth0 supports numerous workflows, web based workflows using Auth0 [Unive
+<% } %>
+<% if (platform==="b2b") { %>
+* Can your users sign up using their organization's identity provider?
+* Can your users be invited or self register?
```
```suggestion
* Can my users be invited or self register?
```

tamigoodall replied:
Nope

mostekcm commented on articles/architecture-scenarios/implementation/b2b/b2b-authentication.md:
```diff
@@ -0,0 +1,133 @@
+
+<%= include('../../_includes/_authentication/_introduction.md', { platform: 'b2b' }) %>
+
+Often companies need to segregate their users by organization and sometimes users can have access to more than one organization.  Knowing which of these scenarios is relevant to your company will help define how to determine in which connection a user exists: whether you need to do it, when you need to do it, and how to accomplish it.  See [Home Realm Discovery](#home-realm-discovery) to determin ...
```
We should eventually move this to the _introduction include to be consistent, but that seems too painful to do right now.

tamigoodall replied:
Done

mostekcm commented on articles/architecture-scenarios/implementation/b2b/b2b-authentication.md:
```diff
@@ -0,0 +1,133 @@
+With enterprise connection support, user identities and credentials are managed by the identity provider of your customers' organization, as well as certain identity claims - which Auth0 will use to populate the user [profile](/architecture-scenarios/implementation/b2b/b2b-profile-mgmt).
+
+::: panel Best Practice
+"Bring your own identity" is a great feature to provide, but if you don't support this from day one, and sometimes even if you do, you may have an organization that wants to switch to their own IDP after already having used the application for a while.  You will need a way to [Link User Accounts](https://auth0.com/docs/link-accounts) (a.k.a. Account Linking) to provide an effective way of associa ...
```
```suggestion
"Bring your own identity" is a great feature to provide, but if you don't support this from day one, and sometimes even if you do, you may have an organization that wants to switch to their own IDP after already having used the application for a while.  You will need a way to [Link User Accounts](/link-accounts) (a.k.a. Account Linking) to provide an effective way of associating the new identity with the old database identity.
```

mostekcm commented on articles/architecture-scenarios/_includes/_authentication/_introduction.md:
```diff
@@ -1,28 +1,38 @@
 
 Auth0 [Universal Login](#universal-login) provides users with a safe and secure experience - no matter whether you choose to provide for user ID/password credentials sign in, or allow the so-called Bring Your Own Identity scenarios provided via [Social Login](https://auth0.com/learn/social-login/). There are also brand recognition benefits to centralizing the login experience with Universal Login ...
 
 Allowing users to sign in via user ID/password credentials means that you're not reliant on the status of third-party identity providers for your users to access your system. You also have the means require the credentials used to align with your corporate policies. Auth0 assists with this by providing you with multiple options in support of user ID/password logins, and the [guidance provided](#u ...
```
```suggestion
Allowing users to sign in via user ID/password credentials means that you're not reliant on the status of third-party identity providers for your users to access your system. You also have the means require the credentials used to align with your corporate policies. Auth0 assists with this by providing you with multiple options in support of user ID/password logins, and the [guidance provided](#username-and-password-authentication) will help you understand you can leverage these options. Adding [social](#social-authentication) support at some stage, as an additional primary authentication factor, gives you added flexibility and can help you better understand your users without the need to question them further by leveraging the information already stored by the various social login [providers](/identityproviders#social).
```

mostekcm commented on articles/architecture-scenarios/_includes/_authentication/_m2m.md:
```diff
@@ -0,0 +1,14 @@
+* A privileged API that may need to be called before a user has authenticated (i.e. from a rule or custom DB script in your Auth0 tenant)
+
+::: panel best practice
+Traditionally, a special "service account" would have been created in order to cater for these scenarios: a user with a username and password that was configured for services which supported non-interactive use cases. That is no longer a recommended approach for many reasons, and the current best practice is to use [OAuth 2.0 Client Credentials Grant](https://auth0.com/docs/flows/concepts/client- ...
```
```suggestion
Traditionally, a special "service account" would have been created in order to cater for these scenarios: a user with a username and password that was configured for services which supported non-interactive use cases. That is no longer a recommended approach for many reasons, and the current best practice is to use [OAuth 2.0 Client Credentials Grant](/flows/concepts/client-credentials) in these situations.
```

mostekcm commented on articles/architecture-scenarios/_includes/_authorization/_application-integration.md:
```diff
@@ -8,6 +8,10 @@ Using Rule extensibility, Auth0 allows you to easily [add custom claims to an ID
+
 ### ID Token scopes
 
 [OIDC Scopes](/scopes/current/oidc-scopes) are typically used by an application to obtain consent for authorized access to a user's details during authentication. Each of the pre-defined scopes returns the set of [standard claims](/scopes/current/oidc-scopes#standard-claims) where defined, and as described in the [OIDC specification](https://openid.net/specs/openid-connect-core-1_0.html#StandardC ...
```
```suggestion
[OIDC Scopes](/scopes/current/oidc-scopes) are typically used by an application to obtain consent for authorized access to a user's details during authentication. Each of the pre-defined scopes returns the set of [standard claims](/scopes/current/oidc-scopes#standard-claims) where defined, and as described in the [OIDC specification](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims). The scopes an application requests depend on which user attributes the application needs. Once the requested scopes are authorized by the user, the claims are returned in the ID Token and are also made available via the [/userinfo](/api/authentication#get-user-info) endpoint.
```

mostekcm commented on articles/architecture-scenarios/_includes/_authorization/_m2m.md:
```diff
@@ -0,0 +1,13 @@
+* A privileged API that may need to be called before a user has authenticated (i.e. from a rule or custom DB script in your Auth0 tenant)
+
+::: panel best practice
+Traditionally, a special "service account" would have been created in order to cater for these scenarios: a user with a username and password that was configured for services which supported non-interactive use cases. That is no longer a recommended approach for many reasons, and the current best practice is to use [OAuth 2.0 Client Credentials Grant](https://auth0.com/docs/flows/concepts/client- ...
```
```suggestion
Traditionally, a special "service account" would have been created in order to cater for these scenarios: a user with a username and password that was configured for services which supported non-interactive use cases. That is no longer a recommended approach for many reasons, and the current best practice is to use [OAuth 2.0 Client Credentials Grant](/flows/concepts/client-credentials) in these situations.
```

mostekcm commented on articles/architecture-scenarios/implementation/b2b/b2b-authentication.md:
```diff
@@ -0,0 +1,133 @@
+If your application already knows what connection (IDP) the user needs, then pass that along when you redirect the user to /authorize using the connection query parameter.
+:::
+
+If this is the case for your application(s) then home realm discovery is a simple matter of storing the Auth0 connection name with the organization specific application configuration and sending that connection name as a parameter when redirecting the user for Universal Login. Sending the connection parameter can be achieved by adding it as a query parameter when you redirect them to the authoriz ...
```
```suggestion
If this is the case for your application(s) then home realm discovery is a simple matter of storing the Auth0 connection name with the organization specific application configuration and sending that connection name as a parameter when redirecting the user for Universal Login. Sending the connection parameter can be achieved by adding it as a query parameter when you redirect them to the authorize endpoint. For more information see the [Authentication API docs](/api/authentication#authorization-code-flow); however, you will generally accomplish this using the SDK for whichever language your application is written in.
```

mostekcm commented on articles/architecture-scenarios/implementation/b2b/b2b-authentication.md:
```diff
@@ -0,0 +1,133 @@
+
+## Enterprise Login
+
+The “bring your own identity” scenario has become a must-have for almost all B2B applications.  Most enterprise companies expect to be able to integrate their IDP into your application so their employees don't need to store another set of credentials.  This is a valuable way of simplifying the user authentication experience without compromising security, and using [Universal Login](#universal-log ...
```
```suggestion
The “bring your own identity” scenario has become a must-have for almost all B2B applications.  Most enterprise companies expect to be able to integrate their IDP into your application so their employees don't need to store another set of credentials.  This is a valuable way of simplifying the user authentication experience without compromising security, and using [Universal Login](#universal-login) makes it easy to start adding support for [Enterprise Connections](/identityproviders#enterprise) with minimal disruption.
```

mostekcm commented on articles/architecture-scenarios/implementation/b2b/b2b-branding.md:
```diff
@@ -0,0 +1,69 @@
+
+## Branding login by organization
+
+Whether or not you need to do special customization on the Universal Login page is determined by how you plan to manage your customers’ organizations.  Before reading through this section, make sure you have read through the [Universal Login section](#universal-login-and-login-pages) and know how you are approaching organizations by reviewing [Multiple Organization Architecture](https://drive.goo ...
```
```suggestion
Whether or not you need to do special customization on the Universal Login page is determined by how you plan to manage your customers’ organization.  Before reading through this section, make sure you have read through the [Universal Login section](#universal-login-and-login-pages) and know how you are approaching organizations by reviewing [Multiple Organization Architecture](https://drive.google.com/a/auth0.com/file/d/1y2G8RNHTBujcCrnMRhp6_phQiRAkZzfF/view?usp=sharing).
```

mostekcm commented on articles/architecture-scenarios/implementation/b2b/b2b-provisioning.md:
```diff
@@ -0,0 +1,131 @@
+## Provisioning organizations
+
+::: panel best practice
+What you need to do when provisioning an organization will depend on how organizations are represented in your system.  This can take some time to step back and consider how users of those organizations will be interacting with your applications.  Please see our [Multiple Organization Architecture documents](https://drive.google.com/a/auth0.com/file/d/1y2G8RNHTBujcCrnMRhp6_phQiRAkZzfF/view?usp=sh ...
```
We should move this document too the docs repo too.

tamigoodall replied:
Done

mostekcm reviewed (changes requested):
Couple of changes

mostekcm commented on articles/architecture-scenarios/implementation/b2b/b2b-authentication.md:
```diff
@@ -0,0 +1,133 @@
+
+## Planning
+
+<%= include('../../_includes/_planning.md'), { platform: 'b2b' }) %>
```
```suggestion
<%= include('../../_includes/_planning.md', { platform: 'b2b' }) %>
```

mostekcm committed:
Apply suggestions from code review

This is a compiler error on the docs repo

tamigoodall committed:
Architecture copyedits

Architecture copyedits

tamigoodall committed:
Merge branch 'b2b-phase-1' of github.com:auth0/docs into b2b-phase-1

mostekcm commented on articles/architecture-scenarios/_includes/_base-ways-to-integrate.md:
```diff
@@ -0,0 +1,3 @@
+There are many different ways Auth0 can be integrated into the <% if (platform === "b2c") { %>CIAM<% } else { %>B2B IAM<% } %> project architecture. Auth0's flexibility comprehensively supports many different use cases, however keep in mind that not every project requires 100% of the capabilities provided by Auth0.
+
+When you embark on your journey to integrate with Auth0, there are many things for you to consider. Knowing what, when, and how best to implement something will help you focus on completing the necessary tasks at the right time. To help you with this, we've put together some [planning guidance](https://drive.google.com/a/auth0.com/file/d/1lQU-uPTfwEi58WJMKL2azUhZhvy9h1IU/view?usp=sharing) that de ...
```
```suggestion
When you embark on your journey to integrate with Auth0, there are many things for you to consider. Knowing what, when, and how best to implement something will help you focus on completing the necessary tasks at the right time.  

<%= include('./_planning.md', { platform: platform }) %>
```

tamigoodall committed:
Final copy edit

Final copy edit

tamigoodall committed:
Apply suggestions from code review

Co-Authored-By: Carlos Mostek <carlos@monthlymealplanner.com>

tamigoodall committed:
Minor fixes

Added stray paragraph to authentication introduction
Added Multitenancy PDF and link to provisioning

tamigoodall committed:
Merge branch 'b2b-phase-1' of github.com:auth0/docs into b2b-phase-1

tamigoodall committed:
Removed stray include link

Removed stray include link
